### PR TITLE
Allow disabling ImageOptim for specific configurations

### PR DESCRIPTION
**Pull request by alexey-pelykh**
- [ ] I have added or updated the specs/tests.
- [ ] I have verified that the specs/tests pass on my computer.
- [ ] I have not attempted to bump, or alter versions.
- [x] This is a documentation change.
- [x] This is a source change.

## Description

Allow to disable ImageOptim for development and CI builds

### CODE REVIEW

File: lib/jekyll/assets/plugins/proxy/optim.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ def initialize
 
         def process
           optimc = @env.asset_config[:plugins][:img][:optim]
+          return @file if optimc == false
```
Comment (envygeeks) [inline lib/jekyll/assets/plugins/proxy/optim.rb]: I think I'd be okay with just a `return @file if !optimc` that way nil disables it too. I never really intended it to be the default, in any circumstance so nil should really disable it too no? But we have a bigger problem if it's running by default.

**Comment by envygeeks** [comment]
Note: This'll also need to be backported, as master is dirty and unstable.

**Comment by alexey-pelykh** [comment]
This patch works well with v3.0.11 w/o anything special other than cherry-pick

File: lib/jekyll/assets/plugins/proxy/optim.rb (hunk 1 above)
Comment (alexey-pelykh) [reply lib/jekyll/assets/plugins/proxy/optim.rb]: Updated

**Comment by envygeeks** [comment]
Thanks 💖

**Comment by muffinresearch** [comment]
@envygeeks hi, do you have an idea of when you think this fix will make it into a stable release?

**Comment by envygeeks** [comment]
I’ve not had time to think about it but I’ll take a look sometime in the next few days and see if I can cut a clean release.

**Pull request merged**